sdk/core/azure-core-test/pom.xml

```diff
@@ -110,6 +116,7 @@
                 <include>io.projectreactor:reactor-test:[3.4.6]</include> <!-- {x-include-update;io.projectreactor:reactor-test;external_dependency} -->
                 <!-- special allowance for azure-core-test as it is not a shipping library: -->
                 <include>org.junit.jupiter:junit-jupiter-api:[5.7.2]</include> <!-- {x-include-update;org.junit.jupiter:junit-jupiter-api;external_dependency} -->
+                <include>org.junit.jupiter:junit-jupiter-params:[5.7.2]</include> <!-- {x-include-update;org.junit.jupiter:junit-jupiter-params;external_dependency} -->
```

Comment:
>include [](http://example.com/codeflow?start=17&length=7)

Why the banned list?

Reply:
This banned list is allowing for the inclusion of using this dependency, previously it was used as a testing dependency which isn't validated but moving it to runtime results in it being validated.

Reply:
I meant why are these libraries in the banned list? Do they have any known vulnerabilities?

Reply:
Ah! No, this is tooling which just requires you to state which dependencies you directly allow in a POM